Code/Tools/AssetProcessor/native/resourcecompiler/JobsModel.cpp

```diff
@@ -355,7 +355,33 @@ namespace AssetProcessor
             AssetProcessor::AssetDatabaseConnection assetDatabaseConnection;
             assetDatabaseConnection.OpenDatabase();
 
-            auto jobsFunction = [this, &assetDatabaseConnection](AzToolsFramework::AssetDatabase::JobDatabaseEntry& entry)
+            // Get historical ProcessJob stats from asset database
+            AZStd::unordered_map<QueueElementID, AZ::s64> historicalStats;
+            auto statsFunction = [&historicalStats](AzToolsFramework::AssetDatabase::StatDatabaseEntry entry)
+            {
+                AZStd::vector<AZStd::size_t> delimiterPos;
+                delimiterPos.reserve(3);
+                for (size_t pos = 0;
+                     (pos = entry.m_statName.find(',', delimiterPos.empty() ? 0 : (delimiterPos.back() + 1))) != AZStd::string::npos;)
+                {
+                    delimiterPos.push_back(pos);
+                }
+                if (delimiterPos.size() != 3)
+                {
+                    return true;
+                }
```

Comment:
I think you could probably use AZ::StringFunc::Tokenize here instead

Reply:
that's the thing I was looking for 😄 updated.